app/views/templates/_history.html.erb

```diff
@@ -0,0 +1,25 @@
+<% unless history.empty? %>
+  <% history.each do |audit| %>
+    <% next unless audit_template? audit %>
```

Comment:
since you're touching this area, would you mind fixing this annoying bug that when the only audit in `@history` is not [template change](https://github.com/theforeman/foreman/blob/develop/app/helpers/audits_helper.rb#L71)? In such case we get an empty tab instead of the alert. I suppose this `next` should be moved to [controller](https://github.com/theforeman/foreman/blob/develop/app/controllers/templates_controller.rb#L137) where we'd perform `select` on `@history`